fink_filters/filter_rate_based_kn_candidates/filter.py

```diff
@@ -0,0 +1,305 @@
+# Copyright 2019-2021 AstroLab Software
+# Authors: Julien Peloton, Juliette Vlieghe
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from pyspark.sql.functions import pandas_udf, PandasUDFType
+from pyspark.sql.types import BooleanType
+
+import numpy as np
+import pandas as pd
+import datetime
+import requests
+import os
+import logging
+
+from astropy.coordinates import SkyCoord
+from astropy.coordinates import Angle
+from astropy import units as u
+from astropy.time import Time
+
+from fink_science.conversion import dc_mag
+
+
+@pandas_udf(BooleanType(), PandasUDFType.SCALAR)
+def rate_based_kn_candidates(
+        objectId, rfscore, snn_snia_vs_nonia, snn_sn_vs_all, drb,
+        classtar, jdstarthist, ndethist, cdsxmatch, ra, dec, ssdistnr, cjdc,
+        cfidc, cmagpsfc, csigmapsfc, cmagnrc, csigmagnrc, cmagzpscic,
+        cisdiffposc
+        ) -> pd.Series:
+    """
+    Return alerts considered as KN candidates.
```

Comment:
I would expand a bit more the documentation as we have different filters. You can perhaps cite the paper from which these cuts were extracted?